### PR TITLE
chore(cypress): remove Cypress from Carbon

### PR DESCRIPTION
**Pull request by nineteen88**
### Proposed behaviour

Remove anything related to Cypress from Carbon

### Current behaviour

Currently we still have some leftover aspects of Cypress within Carbon. With Cypress no longer being utilised by Carbon, it would make sense to remove it from the project.

### Checklist

- [x] Commits follow our style guide

#### QA

Not applicable

### CODE REVIEW

**Comment by nineteen88** [comment]
There are still some comments with references to Cypress within the project. I would suggest we make another ticket to investigate these sooner rather than later and resolve any issues that remain.

**Comment by Parsium** [comment]
**question**: I noticed the CI workflow `forked-ci.yml` [uses a Cypress named token when triggering Chromatic](https://github.com/Sage/carbon/blob/remove-cypress/.github/workflows/forked-ci.yml#L22). Looks like this has been the codebase for quite some time. Wonder if this may be a mistake or not?

File: package.json [hunk 1]
```diff
@@ -30,10 +29,9 @@
     "commit": "git-cz",
     "generate-metadata": "node ./scripts/generate_metadata/index.mjs",
     "type-check:carbon": "tsc --noEmit",
-    "type-check:cypress": "tsc -p ./cypress/tsconfig.json",
     "type-check:playwright": "tsc -p ./tsconfig-playwright.json",
-    "type-check": "npm run type-check:carbon && npm run type-check:cypress && npm run type-check:playwright",
-    "type-check:watch": "concurrently --names 'src,cypress,playwright' -c 'cyan,yellow,magenta' 'npm run type-check:carbon -- --watch' 'npm run type-check:cypress -- --watch' 'npm run type-check:playwright -- --watch'",
+    "type-check": "npm run type-check:carbon && npm run type-check:playwright",
+    "type-check:watch": "concurrently --names 'src,playwright' -c 'cyan,yellow,magenta' 'npm run type-check:carbon -- --watch' 'npm run type-check:playwright -- --watch'",
```
[the remark this answers is not in the document]
Comment (nineteen88) [reply package.json:34]: Ah that was something outside of my knowledge. Appreciate that. I shall get that change in!

**Review by edleeks87 — COMMENTED** [review]
Overall changes look good to me @nineteen88, let's leave the workflow James has pointed out to another ticket. The only other thing I noticed is there's a lot of comments that still reference `cypress`, in most cases I think we can just delete the comment or replace it with `playwright` but I think it makes sense to do it as part of this

File: src/components/message/message.pw.tsx [hunk 2]
```diff
@@ -197,11 +197,7 @@ test.describe("Tests for Message component properties", () => {
     await expect(messagePreview(page)).toHaveCSS("border-radius", "8px");
   });
 
-  // Unable to run this test due to the Message component having a tabIndex of -1.
-  // cypress-plugin-tab does not allow tabbing on elements with an tabIndex of -1,
-  // however this test has been kept incase the bug is ever addressed. https://github.com/kuceb/cypress-plugin-tab/issues/18
-
-  test.skip("should focus icon button when open is true for Message component and tab key is pressed", async ({
+  test("should focus icon button when open is true for Message component and tab key is pressed", async ({
```
Comment (edleeks87) [inline src/components/message/message.pw.tsx:200]: praise: nice!

**Review by edleeks87 — COMMENTED** [review]
happy to approve once @Parsium's comment is resolved

**Review by edleeks87 — APPROVED** [review]

**Review by Parsium — APPROVED** [review]

**Review by divyajindel — APPROVED** [review]

**Pull request merged**